### DISCUSSION TITLE
GPIO as relay output and controll via MQTT

### DISCUSSION THREAD

**Discussion by proasnet** [fredlcore/BSB-LAN#651] [Q&A] [answered]
Dear all,
I am using BSB LAN over five years. Now, I need one GPIO for controll relay (DHW heat) in a electric panel.
Putting another smart relay in the electrical panel seems pointless to me when I already have BSBLAN in the electrical panel.
In the past I saw GPIO settings in config.h, but today I don't see it, so I don't even know if it is supported.
I would imagine controlling it via an MQTT command from the home assistant.
Possible? Please direct me.
Thank you

**Comment by fredlcore** [comment]
Yes, that is still possible. We have only removed the list of allowed pins because the number of possible boards is just too high to keep track of that so users have to do their due diligence. You can access any GPIO using the /G URL command, both for reading and writing. More details are in the manual.

**Comment by proasnet** [comment]
Dear @fredlcore, thank you. I see it in a manual.
Format is `/G<x>=<y>`
Please, on Arduino Due, for example pin 50 to high, format will be as next?
```
/G50=1 or
/G 50=1 or
/G GPIO50=1
```

Thank you very much.

**Comment by fredlcore** [comment, marked as answer]
Only one of your examples meets the format you quoted above ;).
